web/src/popover_menus.js

```diff
@@ -234,6 +234,74 @@ export function toggle_message_actions_menu(message) {
     return true;
 }
 
+function set_compose_box_schedule(element) {
+    const send_later_in = element.id;
+    const send_later_class = element.classList[0];
+    const date = new Date();
+    const selected_send_at_time = scheduled_messages.get_send_at_time_from_opts(
+        send_later_in,
+        send_later_class,
+        date,
+    );
+    return selected_send_at_time;
+}
```

Comment:
This is a rebase conflict resolution error; basically reverting 5113f4e2e96dca2e55ef5b251295c35d00e8684c. You revert it back in the next commit, so it's not visible in the final diff, but this sort of mistake is a reason why to always review your individual commits (either via a graphical viewer like `gitk` or on GitHub after every time you resolve merge conflicts when rebasing -- it's really easy to do the wrong thing.

Reply:
Oops sorry idk how that happened thanks for pointing that out, I will be extra careful from now when resolving commits.